src/esportsbot/lib/client.py

```diff
@@ -19,15 +20,23 @@ class EsportsBot(commands.Bot):
 
     :var reactionMenus: A associating integer menu message IDs to ReactionMenu objects.
     :vartype reactionMenus: ReactionMenuDB
+    :var unknownCommandEmoji: The emote which the bot should react to messages with, if the message attempts to call an unknown command
+    :vartype unknownCommandEmoji: Emote
+    :var STRINGS: A dict-like object containing *all* user facing strings. The first level associates string category names with the second level.
+                    The second level is another dict-like object, mapping string names to parameter-formattable user-facing strings.
+    :vartype STRINGS: str
     """
 
-    def __init__(self, command_prefix: str, unknownCommandEmoji: Emote, **options):
+    def __init__(self, command_prefix: str, unknownCommandEmoji: Emote, userStringsFile: str, **options):
         """
         :param str command_prefix: The prefix to use for bot commands when evoking from discord.
+        :param Emote unknownCommandEmoji: The emote which the bot should react to messages with, if the message attempts to call an unknown command
+        :param str userStringsFile: A path to the `user_strings.toml` configuration file containing *all* user facing strings
         """
         super().__init__(command_prefix, **options)
         self.reactionMenus = ReactionMenuDB()
         self.unknownCommandEmoji = unknownCommandEmoji
+        self.STRINGS: MutableMapping[str, MutableMapping[str, str]] = toml.load(userStringsFile)
```

Comment:
If we decide to nest our strings more (could be useful for the larger cogs), this type annotation will become incorrect